### PR TITLE
[MTDSA-5551] Retrieve Relief Investments Models

### PR DESCRIPTION
**Pull request by Mbrookesy**

### CODE REVIEW

**Comment by Mbrookesy** [comment]
Currently in discussions on whether the models should be seperate or not

**Comment by Mbrookesy** [comment]
Decided to be separate, all good to be reviewed!

File: test/v1/models/response/retrieveReliefInvestments/CommunityInvestmentItemSpec.scala [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      |  "reliefClaimed": 1334.00
+      |}
+        """.stripMargin
+  )
+
+
+  "reads" when {
+    "passed valid JSON" should {
+      "return a valid model" in {
+        communityInvestmentItem shouldBe json.as[CommunityInvestmentItem]
```
Comment (Thepworth95) [inline test/v1/models/response/retrieveReliefInvestments/CommunityInvestmentItemSpec.scala:47]: these should be the other way around, the logic should always be on the left and what it ends up as should be on the right

File: test/v1/models/response/retrieveReliefInvestments/EisSubscriptionsItemSpec.scala [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      |  "reliefClaimed": 43432.00
+      |}
+        """.stripMargin
+  )
+
+
+  "reads" when {
+    "passed valid JSON" should {
+      "return a valid model" in {
+        eisSubscriptionsItem shouldBe json.as[EisSubscriptionsItem]
```
Comment (Thepworth95) [inline test/v1/models/response/retrieveReliefInvestments/EisSubscriptionsItemSpec.scala:49]: same as above

File: test/v1/models/response/retrieveReliefInvestments/RetrieveReliefInvestmentsBodySpec.scala [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+      |  ]
+      |}
+        """.stripMargin
+  )
+
+
+  "reads" when {
+    "passed valid JSON" should {
+      "return a valid model" in {
+        retrieveReliefInvestmentsBody shouldBe json.as[RetrieveReliefInvestmentsBody]
```
Comment (Thepworth95) [inline test/v1/models/response/retrieveReliefInvestments/RetrieveReliefInvestmentsBodySpec.scala:119]: again same as above

File: test/v1/models/response/retrieveReliefInvestments/SeedEnterpriseInvestmentItemSpec.scala [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      |  "reliefClaimed": 3432.00
+      |}
+        """.stripMargin
+  )
+
+
+  "reads" when {
+    "passed valid JSON" should {
+      "return a valid model" in {
+        seedEnterpriseInvestmentItem shouldBe json.as[SeedEnterpriseInvestmentItem]
```
Comment (Thepworth95) [inline test/v1/models/response/retrieveReliefInvestments/SeedEnterpriseInvestmentItemSpec.scala:47]: same here

File: test/v1/models/response/retrieveReliefInvestments/SocialEnterpriseInvestmentItemSpec.scala [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      |  "reliefClaimed": 1334.00
+      |}
+        """.stripMargin
+  )
+
+
+  "reads" when {
+    "passed valid JSON" should {
+      "return a valid model" in {
+        socialEnterpriseInvestmentItem shouldBe json.as[SocialEnterpriseInvestmentItem]
```
Comment (Thepworth95) [inline test/v1/models/response/retrieveReliefInvestments/SocialEnterpriseInvestmentItemSpec.scala:47]: same here

File: test/v1/models/response/retrieveReliefInvestments/VctSubscriptionsItemSpec.scala [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+      |  "reliefClaimed": 1334.00
+      |}
+        """.stripMargin
+  )
+
+
+  "reads" when {
+    "passed valid JSON" should {
+      "return a valid model" in {
+        vctSubscriptionsItem shouldBe json.as[VctSubscriptionsItem]
```
Comment (Thepworth95) [inline test/v1/models/response/retrieveReliefInvestments/VctSubscriptionsItemSpec.scala:47]: and here

**Pull request merged**